media/js/app/panel.js

```diff
@@ -57,23 +68,6 @@
             jQuery(self.el).css('height', visible + 'px');
         };
 
```

Comment:
The assignment.feature, scenario 2 selenium failure is related to the removal of the loadTemplates function. In panel.js at Ln. 261, self.loadTemplates is still referenced in the newPanel function. (https://github.com/ccnmtl/mediathread/blob/mustache-collection-conversion/media/js/app/panel.js#L261). 

newPanel is a cute little way to automagically open an Assignment Response panel when a student clicks "Respond to Assignment" within the assignment space. If this is a pain to fix, replacing this logic with a simple redirect is a completely fine option.